### PR TITLE
Feature/fix bug regular expression

### PR DESCRIPTION
**Pull request by Nikita-Mamaisur**

### CODE REVIEW

File: src/Extensions/Extensions.TurnContext/Extensions.EncodeTextWithStyle.cs [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        var encodedString = source;
+
+        if (style == BotTextStyle.Default)
+        {
+            foreach (var regItem in RegexReplacementWithDefaultStyle)
+            {
+                encodedString = regItem.Key.Replace(encodedString, regItem.Value);
+            }
+        }
+
```
Comment (pmosk) [inline src/Extensions/Extensions.TurnContext/Extensions.EncodeTextWithStyle.cs]: Remove empty line

File: src/Extensions/Extensions.TurnContext/Extensions.EncodeTextWithStyle.cs [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        if (turnContext is null || turnContext.IsNotTelegramChannel())
+        {
+            return source;
+        }
+
+        var encodedString = source;
+
+        if (style == BotTextStyle.Default)
```
Comment (pmosk) [inline src/Extensions/Extensions.TurnContext/Extensions.EncodeTextWithStyle.cs]: It's better to use "is" instead "=="

File: src/Extensions/Extensions.TurnContext/Extensions.EncodeTextWithStyle.cs (hunk 1 above)
Comment (Nikita-Mamaisur) [reply src/Extensions/Extensions.TurnContext/Extensions.EncodeTextWithStyle.cs]: Fixed

File: src/Extensions/Extensions.TurnContext/Extensions.EncodeTextWithStyle.cs (hunk 2 above)
Comment (Nikita-Mamaisur) [reply src/Extensions/Extensions.TurnContext/Extensions.EncodeTextWithStyle.cs]: Fixed

**Review by Sauric — APPROVED** [review]

**Review by pmosk — APPROVED** [review]

**Review by Sauric — APPROVED** [review]

**Pull request merged**